package.json

```diff
@@ -46,6 +50,7 @@
     "test": "npm-run-all test:*",
     "test:karma": "karma start",
     "test:lint": "eslint .",
-    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec"
+    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec",
+    "clientize": "webpack"
```

Comment:
This isn't ideal since it doesn't do any compressing. Richard suggested using:

`webpack --optimize-minimize --optimize-dedupe`

But when I run it here it throws some errors I have to dig into...